app/javascript/flavours/polyam/features/emoji/emoji_compressed.js

```diff
@@ -9,7 +9,7 @@
 
 // This version comment should be bumped each time the emoji data is changed
 // to ensure that the prevaled file is regenerated by Babel
-// version: 2
+// version: 3
```

Comment:
Technically, this should be "4", but that comment was added after polyam-glitch started using custom data.